rotkehlchen/db/upgrades/v38_v39.py

```diff
@@ -261,6 +262,20 @@ def _add_arbitrum_one_location_and_nodes(write_cursor: 'DBCursor') -> None:
     log.debug('Exit _add_arbitrum_one_location_and_nodes')
 
 
+def _remove_saddle_excahnge(write_cursor: 'DBCursor') -> None:
+    log.debug('Enter _remove_saddle_excahnge')
+    write_cursor.execute('SELECT value FROM settings WHERE name="current_price_oracles"')
+    if (data := write_cursor.fetchone()) is None:
+        return  # oracles not configured
+    oracles: list[str] = json.loads(data[0])
```

Comment:
It should never happen here, but please since this is an upgrade and is critical code can you please handle json decoding error? 

In case it happens, log a critical error and proceed to remove the setting. I think this would be the only sensible option?